forslag.md

```diff
@@ -2,67 +2,62 @@
-Så hvis bilen gik i stykker, ville du være tvunget til at køre den ødelagte bil hjem Asien, hvor den blev bygget.
+Tænk hvis det også gjaldt for din bil: Så ville producenten gøre teknikken i bilen utilgængelig for selv din mekaniker, og du ville derfor være helt afgængig af producenten til at reparere bilen igen.
 
 Desværre er det præcis det, der sker med vores offentlige IT.
-Alene i årene 2000-2014 har staten Danmark kastet 2,902 milliarder kroner væk på skrottet IT.<sup><a name="dr8">2</a></sup>
-Det svarer til 585 fuldtidsansatte sygeplejersker i de 14 år.
+Alene i årene 2000-2014 har staten Danmark brugt 2902 millioner kroner på skrottet IT.<sup><a name="dr8">2</a></sup>
+Det svarer til 585 fuldtidsansatte sygeplejersker i 14 år.
 
 Hvis systemerne havde været ejet af staten kunne vi &ndash; ligesom med en ødelagt bilmotor &ndash; redde og genbruge store dele af dem, selv efter de var brudt sammen.
 
-Det er en langt bedre løsning på en bedre digital hverdag i Danmark, fordi det vil gøre os til herre over eget hus. Vi skal kræve
+Det er en langt bedre løsning på en bedre digital hverdag i Danmark, og vi skal derfor kræve
 at vores leverandører lever op til åbne standarder så vi kan bygge videre på vores infrastruktur.
 
 ### Åbenhed giver kvalitet
 Millioner af sygdomsforløb, straffeattester og andre fortrolige
-dokumenter bliver dagligt sendt igennem vores offentlige IT platforme. Men langt størstedelen af vores IT infrastruktur er udenfor vores kontrol.
-
-Tag Danmarks love som et eksempel: de sørger for at staten og borgerne opfører sig ordentligt og de skaber tryghed og livskvalitet.
-Fordi vores IT systemer er lukkede kan vi ikke se om de overholder vores love.
-Vi kan faktisk ikke se om det er kvalitet eller skrammel.
-Hvorfor har vi ikke love, der sikrer at vores offentlige IT opfører sig ordentligt?
+dokumenter bliver dagligt sendt igennem vores offentlige IT platforme. Men langt størstedelen af vores IT infrastruktur er udenfor vores kontrol, da systemerne er lukkede. Dette bevirker, at koden bag systemerne ikke kan kvalitetssikres og det beværliggører kontrol med om koden overholder gældende love.
 
 Faktisk har uafhængig forskning gentagne gange vist at åbent IT ikke kun er billigere, men også bedre og mere effektivt <sup>[3](#samoladas)</sup>.
 Forklaringen ligger i at åben kode bliver bygget og set af mange flere mennesker. Og fire øjne er som bekendt bedre end to.
 
 ## Åbenhed giver innovation og vækst
 Nye IT-systemer bliver typisk købt i udlandet, fremfor at blive produceret
-herhjemme. I stedet for at *forbruger* IT, burde vi fokusere mere på
-at *producere* IT. Hvis vi holder fast i åbenheden skaber vi kvalitet til vores
-præcise behov, fremfor at købe dyre og dårlige metervarer.
+herhjemme. Selv om Danmark investerer massivt i IT systemer, skulle vi fokusere mere på at producere IT, frem for at forbruge IT. Hvis vi holder fast i åbenheden skaber vi kvalitet som matcher vores
```

Comment:
Suggestion: "på at producere IT" -> "på vores produktion af IT"?

Det er lidt uklart for mig præcis hvad der ligger i "producere IT" (det var det også i det oprindelige). Er det at vi skal fokusere på at blive bedre til at producere IT i det offentlige, i staten? Eller at vi skal gå aktivt ind og sætte nogle standarder for hvordan den produceres? Eller ... der er mange måder at læse det på, synes jeg. Og jeg er ikke helt sikker på hvad der ligger i det.

Reply:
Jeg kan godt se hvad du mener med at det er uklart. Hvad var den oprindelige tanke bag at skrive hvad der stod? For jeg synes sagtens der kunne være en pointe som er værd at have med, men måske skal den bare omformuleres eller gøres mere konkret?

Reply:
Enig. Måske @Jegp kan give sit besyv med.